msi/GitUI_en-us.wxl

```diff
@@ -23,15 +23,15 @@
   <String Id="GitChooseEnvironmentDlg_Title" Overridable="yes">[ProductName] Setup</String>
   <String Id="GitChooseEnvironmentDlgBannerBitmap" Overridable="yes">WixUI_Bmp_Banner</String>
   <String Id="GitChooseEnvironmentDlg_RbBash" Overridable="yes">Only use Git from Git from the Bash &amp;shell</String>
-  <String Id="GitChooseEnvironmentDlg_RbBoth" Overridable="yes">Use Git from &amp;both the Windows Command Prompt and the Bash shell</String>
-  <String Id="GitChooseEnvironmentDlg_RbCmd" Overridable="yes">Use Git and optional Unix tools from the Windows Command &amp;Prompt</String>
+  <String Id="GitChooseEnvironmentDlg_RbCmd" Overridable="yes">Use Git from &amp;both the Windows Command Prompt and the Bash shell</String>
+  <String Id="GitChooseEnvironmentDlg_RbCmdTools" Overridable="yes">Use Git and optional Unix tools from the Windows Command &amp;Prompt</String>

```

Comment:
Per a previous comment, sometimes the journey is more important than the destination. But, in this case, I think you're right. The commit history would be a bit cleaner and easier to follow if this change "appeared" to be the original UI implementation, and then this commit wires everything up and fills in the details.